### PR TITLE
[14.0] sale_order_import_ubl: import line notes

### PR DESCRIPTION
**Pull request by simahawk**
See atomic commits for details

### CODE REVIEW

File: sale_order_import/wizard/sale_order_import.py [hunk 1]
```diff
@@ -470,6 +470,13 @@ def _prepare_create_order_line(
             vals["order_id"] = order
             vals = solo.play_onchanges(vals, ["product_id"])
             vals.pop("order_id")
+
+        # Handle additional fields dynamically if available.
+        # This way, if you add a field to a record and it's value is injected by a parser
+        # you won't have to override `_prepare_create_order_line` to let it propagate.
+        for k, v in import_line.items():
+            if k not in vals and k in solo._fields:
+                vals[k] = v
```
Comment (florentx) [inline sale_order_import/wizard/sale_order_import.py:479]:
```suggestion
        for field in import_line:
            if field not in vals and field in solo._fields:
                vals[field] = import_line[field]
```

**Review by florentx — APPROVED** [review]
LG

File: sale_order_import/wizard/sale_order_import.py (hunk 1 above)
Comment (simahawk) [reply sale_order_import/wizard/sale_order_import.py:479]: I don't get how this is better? :thinking:

**Review by sebalix — APPROVED** [review]

**Pull request merged**